src/js/references/components/CreateReference.tsx

```diff
@@ -1,28 +1,30 @@
 import React from "react";
 import { useLocation } from "react-router-dom";
 import { ContainerNarrow, Tabs, TabsLink, ViewHeader, ViewHeaderTitle } from "../../base";
-import EmptyReference from "./Empty";
+import EmptyReference from "./EmptyReference";
 import { ImportReference } from "./ImportReference";
 
+type LocationState = {
+    emptyReference: boolean;
+    importReference: boolean;
+};
+
+/**
+ * The create reference view with options to create an empty reference or import a reference
+ */
 export function CreateReference() {
-    const location = useLocation();
+    const location = useLocation<LocationState>();
 
     return (
         <ContainerNarrow>
             <ViewHeader title="Create Reference">
                 <ViewHeaderTitle>Create Reference</ViewHeaderTitle>
             </ViewHeader>
             <Tabs>
-                <TabsLink
-                    to={{ state: { emptyReference: true } }}
-                    isActive={(_, location) => location.state.emptyReference}
-                >
+                <TabsLink to={{ state: { emptyReference: true } }} isActive={() => location.state.emptyReference}>
```

Comment:
```suggestion
                <TabsLink to={{ state: { emptyReference: true } }} isActive={location.state.emptyReference}>
```
Can you just exclude the arrow functions? Seems like the kind of thing you should be able to do, but depends on TabsLink api.

Reply:
No it seem as if the TabsLink api requires an arrow function